doc/modules/changes/20170510_mfraters

```diff
@@ -0,0 +1,6 @@
+Added functions to compute weighted p norm averages and the derivatives
+of these weighted p norm averages in utilites. The derivatives function 
+is templated to be able to compute derivatives which are doubles and 
+tensors.
```

Comment:
can you reference the full name of the functions somewhere, as in `Utilities::name_of_function` so that it is properly linked to?